project/src/main/java/com/google/sps/servlets/EventCreationServlet.java

```diff
@@ -62,4 +83,31 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Event in database
     response.getWriter().println(CommonUtils.convertToJson(SpannerTasks.getEventById(event.getId()).get().toBuilder().build()));
   }
+
+/*
+ * @param text: String of text that includes event name and description
+ * @return categoryNames: returns selected names of labels that NLP API suggests for text
+*/
+  public ArrayList<String> getNlp(String text) throws IOException {
+    ArrayList<String> categoryNames = new ArrayList<String>();
+
+    // Use Gcloud NLP API to predict labels based on user inputted event name and description
+    try (LanguageServiceClient language = LanguageServiceClient.create()) {     
+    Document doc = Document.newBuilder().setContent(text).setType(Type.PLAIN_TEXT).build();
```

Comment:
Indent the code that goes inside the try block

Reply:
done